config/routes.rb

```diff
@@ -144,8 +145,6 @@
   end
 
   namespace :items do
-    post 'version/close',  action: :close_version, as: 'close_version'
-    post 'version/open',   action: :open_version,  as: 'open_version'
```

Comment:
Just making sure -- is this going to remove argo routes?  If so, we many need to ensure no one uses them ...

Reply:
I don't believe these are used now that we have the member routes above.

Reply:
I searched through Argo and sul-dlss and could find no code using these endpoints.